src/main/webapp/logged_in_homepage.html

```diff
@@ -47,14 +47,31 @@ <h1 class="masthead-heading mb-0">Logged In Homepage</h1>
       <div class="container">
         <h3>Match History:</h3>
         <ul id="match-history"></ul>
-        <div class="form" id="matchForm">
-          <form class="form-container">
-            <h3>Get New Matches</h3>
-            <label for="product_area"><b>Product Area</b></label>
-            <input type="text" placeholder="Enter Product Area" id="product_area" required>
-            <input type="button" class="btn btn-primary" value="Submit" onclick="sendMatchRequest()" data-toggle="modal" data-target="#exampleModal">
-          </form>
-        </div>
+
+       <div class="form" id="matchForm">
+        <form class="form-container" id="match_form">
+        <h3>Get New Matches</h3>
+
+        <label for="specialty"><b>Select a Specialty:</b></label>
+        <select
+          name="specialty"
+          id="specialty"
+          form="match_form"
+          required
+        />
+          <option value="Machine Learning">Maching Learning</option>
+          <option value="Artificial Intelligence">Artificial Intelligence</option>
+          <option value="Systems">Systems</option>
```

Comment:
I take it you plan to populate these option values programatically after the MVP?

Reply:
I had planned on just leaving the values hard-coded, but I can see changing this after the MVP to allow for more custom specialties. Definitely a good idea to consider.